docker/wfm/app/entry-point.sh

```diff
@@ -19,5 +19,21 @@ set -eu ${DEBUG:+-x}
 PATH=${PATH}:/opt/storm/bin
 
 cd /app
-make kill-all
-exec make deploy-all
+
+case ${WFM_TOPOLOGIES_MODE:-} in
+
+  blue)
+    make kill-all-blue
+    exec make deploy-all-blue
```

Comment:
do not use `exec` in this way. In the original code, it was some kind of optimization - we can "close/replace" the existing bash process with make (at the end of the script). Now you have some logic around this `make` call... and should not close the existing bash process too early.

PS it can be done in a simpler way (if WFM_TOPOLOGIES_MODE can contain only "blue" or "green" or are not defined):
```
make kill-all${WFM_TOPOLOGIES_MODE:+-$WFM_TOPOLOGIES_MODE}
exec make deploy-alll${WFM_TOPOLOGIES_MODE:+-$WFM_TOPOLOGIES_MODE}
```